watch subscriptions in loadtest (#20)
kyma-project/eventing-tools: docs/loadtest/README.md

It does that by sending Cloudevents to the Eventing publisher proxy forever and consume them inside Kyma functions.
Ideally, it should be used when JetStream is used as the active Eventing backend.

## Subscriptions

The publisher watches all subscriptions in the cluster.

> To activate sending events for one subscription you have to label that subscription with one of these labels:

Reviewer: ```suggestion
To activate sending events for one subscription, you must label that subscription with one of the following labels:
```

- eventing-loadtest: legacy 
- eventing-loadtest: cloudevent

> The load tester will then create events using the event types configured in the subscription and send them with the format configured in the label.

Reviewer: ```suggestion
After you configured the event type in the label, the load tester creates events accordingly and sends them with the format configured in the label.
```

> The load generated per event type will be extracted from the event type. To do this the event type must match the following pattern:

Reviewer: ```suggestion
The load generated per event type is extracted from the event type. To do this, the event type must match the following pattern:
```

> `<event-name>.v<Number of events per second>`, eg. `order.created.v500` will create events of type order.create.v500 and the sender will try to publish 500 events per second.

Reviewer: ```suggestion
`<event-name>.v<Number of events per second>`. For example, `order.created.v500` creates events of type order.create.v500 and the sender tries to publish 500 events per second.
```

> Note: Encoding the EPS in the event type version is used only for debugging purposes and is not a production use-case.

> There is no special handling of subscriptions otherwise. The `sink` and also the `maxInFlight` settings will be respected.

> This publisher works best in combination with the loadtest-subscriber. Two instances of the subscriber are deployed using the kustomize deployment. Configure the subscriptions to use those by setting the sink to `loadtest-subscriber-0.eventing-test.svc.cluster.local` or `loadtest-subscriber-1.eventing-test.svc.cluster.local`

Reviewer 2: Do we still need to have 2 sinks? Does it make sense to always use the same sink and delete the other?

Author: the two sinks are configured to:
- 1. run on the same node as ec
- 2. run on different node than ec

but i agree... the whole concept of those sinks should be reconsidered
